### PR TITLE
Assignment of DS Session1

### PR DESCRIPTION
**Pull request by meta-shubham-jain**

### CODE REVIEW

File: DS/DS-Session1/Queue/Node.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    public int getData() {
+        return data;
+    }
+
+    /**
+     * It returns reference of node
+     * 
+     * @return
+     */
+    public Node getLink() {
```
Comment (Ramkumar68) [inline DS/DS-Session1/Queue/Node.java]: @meta-shubham-jain getNext is more appropriate than getLink

File: DS/DS-Session1/Queue/QueueUsingArray.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
+/**
+ * It is implementing Queue interface which has methods related to queue and it has
+ * implemented using array. Front is for removal and Rear is for insertion in Queue
+ * 
+ * @author Shubham Jain
+ *
+ */
+public class QueueUsingArray implements Queue {
```
Comment (Ramkumar68) [inline DS/DS-Session1/Queue/QueueUsingArray.java:8]: @meta-shubham-jain 
Queue using array may be more sensible if you user circular array

Comment (meta-shubham-jain) [reply DS/DS-Session1/Queue/QueueUsingArray.java:8]: Circular Queue is in second Assignment to implement

**Review by Ramkumar68 — CHANGES_REQUESTED** [review]
Review changes need to do or explain why not need to be done

**Review by Ramkumar68 — APPROVED** [review]

**Pull request merged**